src/EnergyPlus/SetPointManager.cc

```diff
@@ -6293,16 +6618,16 @@ void DefineOAPretreatSetPointManager::calculate(EnergyPlusData &state)
     // na
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int RefNode;            // setpoint reference node number
-    int MixedOutNode;       // mixed air outlet node number
-    int OAInNode;           // outside air inlet node number
-    int ReturnInNode;       // return air inlet node number
-    Real64 OAFraction;      // outside air fraction of mixed flow rate
-    Real64 ReturnInValue;   // return air inlet node mass flow rate
-    Real64 RefNodeSetPoint; // setpoint at reference node
-    Real64 MinSetPoint;     // minimum allowed setpoint
-    Real64 MaxSetPoint;     // maximum allowed setpoint
-    bool HumiditySetPoint;  // logical to indicate if this is a humidity setpoint
+    int RefNode;                // setpoint reference node number
+    int MixedOutNode;           // mixed air outlet node number
+    int OAInNode;               // outside air inlet node number
+    int ReturnInNode;           // return air inlet node number
+    Real64 OAFraction;          // outside air fraction of mixed flow rate
+    Real64 ReturnInValue = 0;   // return air inlet node mass flow rate
+    Real64 RefNodeSetPoint = 0; // setpoint at reference node
+    Real64 MinSetPoint = 0;     // minimum allowed setpoint
+    Real64 MaxSetPoint = 0;     // maximum allowed setpoint
+    bool HumiditySetPoint;      // logical to indicate if this is a humidity setpoint
```

Comment:
Visual Studio threw some warnings because initial values were not assigned so we updated this part.

Reply:
Ideally, all of these variables should be initialized when declared. Or just declare them below when first used.